ui/logviewer/App.jsx

```diff
@@ -169,11 +169,11 @@ class App extends React.PureComponent {
   render() {
     const {
       job, rawLogUrl, reftestUrl, jobDetails, jobError, jobExists,
-      revision, errors, highlight, jobId,
+      revision, errors, highlight, jobId, repoName,
     } = this.state;
     const extraFields = [{
       title: 'Revision',
-      url: getJobsUrl({ revision, selectedJob: jobId }),
+      url: getJobsUrl({ revision, selectedJob: jobId, repo: repoName }),
```

Comment:
Should we put `repo` before `revision` so that the order is the same as it was before? (in rough order of "specificness")